text/0000-replace-blacklist-whitelist.md

```diff
@@ -72,9 +74,24 @@ There are also other possible terms that could be used to replace `blacklist` an
 - `allowlist` / `denylist`
 - `allowlist` / `disallowlist`
 - `allow` / `deny`
+- `allowed` / `disallowed`
 - `allowedlist` / `disallowedlist`
 - `safelist` / `blockedlist`
 
+For reference, this is how often these terms are used accross the Ember ecosystem according to Ember Observer - all searches done inside of index.js files for easier comparison:
```

Comment:
The total usage isn't super useful, without knowing the context it's hard to tell if the usage and constraints are the same or if the author regrets the choice. Simply cargo culting verbiage would be be unfortunate.

Given the constraints variants of `allow.*` `disallow.*` appear to be the most clear.